api/go/info/info.pb.go

```diff
@@ -804,6 +804,116 @@ func (ZDeviceState) EnumDescriptor() ([]byte, []int) {
 	return file_info_info_proto_rawDescGZIP(), []int{11}
 }
 
+type StorageRaidType int32
+
+const (
+	StorageRaidType_STORAGE_RAID_TYPE_UNSPECIFIED StorageRaidType = 0
+	StorageRaidType_STORAGE_RAID_TYPE_RAID_ZERO   StorageRaidType = 1 //RAID-0
+	StorageRaidType_STORAGE_RAID_TYPE_RAID_MIRROR StorageRaidType = 2 // RAID-1
+	StorageRaidType_STORAGE_RAID_TYPE_RAIDZ       StorageRaidType = 3 // raidz (RAID-5)
+	StorageRaidType_STORAGE_RAID_TYPE_RAIDZ_TWO   StorageRaidType = 4 // raidz2 (RAID-6)
+	StorageRaidType_STORAGE_RAID_TYPE_RAIDZ_THREE StorageRaidType = 5 // raidz3 (RAID-7)
+	StorageRaidType_STORAGE_RAID_TYPE_NORAID      StorageRaidType = 6 // without RAID
+)
+
+// Enum value maps for StorageRaidType.
+var (
+	StorageRaidType_name = map[int32]string{
+		0: "STORAGE_RAID_TYPE_UNSPECIFIED",
+		1: "STORAGE_RAID_TYPE_RAID_ZERO",
+		2: "STORAGE_RAID_TYPE_RAID_MIRROR",
+		3: "STORAGE_RAID_TYPE_RAIDZ",
```

Comment:
Same comment as above. Change this to RAIDZ_ONE

Reply:
i will fix it